### PR TITLE
Updating some SkyDNS templates to v1beta3

### PR DESCRIPTION
**Pull request by krousey**
#8087

### CODE REVIEW

File: cluster/libvirt-coreos/skydns-rc.yaml [hunk 1]
```diff
@@ -1,44 +1,66 @@
...
-                    \"-addr=0.0.0.0:53\",
-                    \"-domain=${DNS_DOMAIN}.\",
-            ]
-            ports:
-              - name: dns
-                containerPort: 53
-                protocol: UDP
+  template:
+    metadata:
+      creationTimestamp: null
```
Comment (vishh) [inline cluster/libvirt-coreos/skydns-rc.yaml]: nit: Remove `creationTimestamp` here and in other places

**Comment by vishh** [comment]
Can we re-use the configs under `cluster/addons/dns/`? `replicas` and `domain` fields have to set to some concrete value in this case.

File: cluster/libvirt-coreos/skydns-rc.yaml (hunk 1 above)
Comment (krousey) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Done

**Comment by krousey** [comment]
I don't know if we can our not. I know these are used by util.sh in the same directory and passed through a rendering function.

**Comment by vishh** [comment]
IIUC, the skydns configs under `cluster/addons/dns` can be used as-is except for the salt configs inside it.

**Comment by krousey** [comment]
Please take another look as it was failing to build because some analytics links were missing from some README files. Is this ok?

File: cluster/libvirt-coreos/skydns-rc.yaml [hunk 2]
```diff
@@ -1,44 +1,61 @@
...
+    metadata:
+      labels:
+        k8s-app: skydns
+    spec:
+      containers:
+      - args:
+        - \"/etcd\"
+        - \"-bind-addr=127.0.0.1\"
+        - \"-peer-bind-addr=127.0.0.1\"
+        capabilities: {}
```
Comment (bgrant0607) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Please remove fields containing default values. The conversion script doesn't do that currently. If the field wasn't specified previously, it probably doesn't need to be specified in v1beta3, either, especially if it's just empty/false/0.

File: examples/limitrange/README.md [hunk 3]
```diff
@@ -1 +1,4 @@
 Please refer to this [doc](https://github.com/GoogleCloudPlatform/kubernetes/blob/620af168920b773ade28e27211ad684903a1db21/docs/design/admission_control_limit_range.md#kubectl).
+
+
+[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/examples/limitrange/README.md?pixel)]()
```
Comment (bgrant0607) [inline examples/limitrange/README.md]: This has been fixed at HEAD and should go away when you rebase.

File: cluster/libvirt-coreos/skydns-rc.yaml [hunk 4]
```diff
@@ -1,44 +1,43 @@
...
+      labels:
+        k8s-app: skydns
+    spec:
+      containers:
+      - args:
+        - \"/etcd\"
+        - \"-bind-addr=127.0.0.1\"
+        - \"-peer-bind-addr=127.0.0.1\"
+        image: quay.io/coreos/etcd:latest
+        imagePullPolicy: Always
```
Comment (bgrant0607) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Why does this have pull policy Always? It didn't before.

File: cluster/libvirt-coreos/skydns-rc.yaml [hunk 5]
```diff
@@ -1,44 +1,43 @@
...
+        name: etcd
+        terminationMessagePath: /dev/termination-log
+      - args:
+        - \"-domain=${DNS_DOMAIN}\"
+        image: kubernetes/kube2sky:1.0
+        imagePullPolicy: IfNotPresent
+        name: kube2sky
+        terminationMessagePath: /dev/termination-log
```
Comment (bgrant0607) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Please eliminate default fields.

**Comment by bgrant0607** [comment]
Ping

cc @lhuard1A

File: cluster/libvirt-coreos/skydns-rc.yaml (hunk 4 above)
Comment (krousey) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Defaults were added in the conversion and this is the default. Removing it.

File: cluster/libvirt-coreos/skydns-rc.yaml (hunk 5 above)
Comment (krousey) [inline cluster/libvirt-coreos/skydns-rc.yaml]: Done.

File: cluster/libvirt-coreos/skydns-rc.yaml [hunk 6]
```diff
@@ -1,44 +1,37 @@
...
+    metadata:
+      labels:
+        k8s-app: skydns
+    spec:
+      containers:
+      - args:
+        - \"/etcd\"
+        - \"-bind-addr=127.0.0.1\"
+        - \"-peer-bind-addr=127.0.0.1\"
+        image: quay.io/coreos/etcd:latest
```
Comment (bgrant0607) [inline cluster/libvirt-coreos/skydns-rc.yaml]: In general, it's clearer to put name, then image, then other fields.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**